src/pytest_inmanta_lsm/__init__.py

```diff
@@ -13,20 +13,21 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def retry_limited(fun: Callable[[], bool], timeout: int) -> None:
+def retry_limited(fun: Callable[[], bool], *, timeout: int, retry_interval: float = 1.0) -> None:
```

Comment:
This is probably better for absolute backwards compatibility :
```suggestion
def retry_limited(fun: Callable[[], bool], timeout: int, *, retry_interval: float = 1.0) -> None:
```